src/overmap_ui.cpp

```diff
@@ -82,6 +84,68 @@ std::tuple<char, nc_color, size_t> get_note_display_info( const std::string &not
     return result;
 }
 
+std::array<std::pair<nc_color, long>, 9> get_overmap_neighbors( const tripoint &current )
```

Comment:
It would be better to define a couple of local static constants: width and height of the neighborhood. This would eliminate the magic numbers and make the code that rely on them more robust.